spritzle/alert.py

```diff
@@ -53,7 +43,7 @@ def __init__(self):
         self.handlers = {
             'all_categories': [debug_handler],
         }
-        self.categories = build_categories()
+        self.categories = lt.alert.category_t.names
```

Comment:
This doesn't exist in my libtorrent:
```
>>> lt.alert.category_t.names
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'category_t' has no attribute 'names'
```

Reply:
Please make sure you build your libtorrent with ```--disable-deprecated-functions```

Reply:
Okay, I'll rebuild and check. The problem with the original implementation was that the alert manager was crashing on the very first alert, trying to do `&` against things that weren't integers. https://github.com/spritzle/spritzle/blob/master/spritzle/alert.py#L88

Reply:
My guess is that is because you don't have --disable-deprecated-functions .. which isn't great actually because most people won't have that so it'd be nice if you could get it working with deprecated functions enabled.